content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Virtual IPs and Service Proxies
+content_type: reference
+---
+
+<!-- overview -->
+Every {{< glossary_tooltip term_id="node" text="node" >}} in a Kubernetes
+cluster runs a [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/)
+(unless you have deployed your own alternative component in place of `kube-proxy`).
+
+The `kube-proxy` component is responsible for implementing a _virtual IP_
+mechanism for {{< glossary_tooltip term_id="service" text="Services">}}
+of `type` other than
+[`ExternalName`](/docs/concepts/services-networking/service/#externalname).
+
+
+A question that pops up every now and then is why Kubernetes relies on
+proxying to forward inbound traffic to backends. What about other
+approaches? For example, would it be possible to configure DNS records that
+have multiple A values (or AAAA for IPv6), and rely on round-robin name
+resolution?
+
+There are a few reasons for using proxying for Services:
+
+* There is a long history of DNS implementations not respecting record TTLs,
+  and caching the results of name lookups after they should have expired.
+* Some apps do DNS lookups only once and cache the results indefinitely.
+* Even if apps and libraries did proper re-resolution, the low or zero TTLs
+  on the DNS records could impose a high load on DNS that then becomes
+  difficult to manage.
+
+Later in this page you can read about how various kube-proxy implementations work.
+Overall, you should note that, when running `kube-proxy`, kernel level rules may be modified
+(for example, iptables rules might get created), which won't get cleaned up, in some
+cases until you reboot.  Thus, running kube-proxy is something that should only be done
+by an administrator which understands the consequences of having a low level, privileged
+network proxying service on a computer.  Although the `kube-proxy` executable supports a
+`cleanup` function, this function is not an official feature and thus is only available
+to use as-is.
+
+<!-- body -->
+
+## Proxy modes
+
+Note that the kube-proxy starts up in different modes, which are determined by its configuration.
+
+- The kube-proxy's configuration is done via a ConfigMap, and the ConfigMap for
```

Comment:
According to the [original text](https://kubernetes.io/docs/concepts/services-networking/service/#configuration) it's a ConfigMap.

Because this PR is similar to the moving-text-around-commit from https://github.com/kubernetes/website/pull/30817, I'm willing to leave this as is. If it's important, I can file an issue about the text here, wait 'til that is fixed, and then revisit this PR.